models/public/mobilenet-yolo-syg/mobilenet-yolo-syg.md

```diff
@@ -0,0 +1,124 @@
+# mobilemet-yolo-syg
+
+## Use Case and High-Level Description
+
+  This is a Keras\* version of `mobilenet-yolov4` model designed to perform real-time vehicle detection
+  on FFmpeg-OpenVINO. It can realize illegal lane change detection and speed detection by developing FFmpeg.
```

Comment:
I think saying that model is trained for real-time vehicle detection is enough. Rest of phrase `on FFmpeg-OpenVINO` may confuse people and let them think that model can be used with FFmpeg only.

Could you please also clarify the following phrase: `It can realize illegal lane change detection and speed detection by developing FFmpeg`. What and why should be developed at FFmpeg side in order to detect lane intersection or speed detection?